ignite/templates/module/create/ibc/testutil/keeper/ibc.go.plush

```diff
@@ -0,0 +1,154 @@
+package keeper
```

Comment:
What is the context of this file? It looks beyond the scope of the PR if we refer to its description

Reply:
It's because of the ibc-go upgrade from 3.0.0 to 3.0.1. 

In ibc-go 3.0.1, they added a check in `NewKeeper` which panics if some of the parameters are `nil` (see https://github.com/cosmos/ibc-go/pull/1284), so some CLI integration tests were failing because of that.

This commit [5e7ca57](https://github.com/ignite/cli/pull/2628/commits/5e7ca57e319d8ec8ffa4f4ad455dde4e7fa640cc) fixes the problem.

The file you pointed is a kind of a helper that gather all the parameters required, the proper fix is in the other file `{{moduleName}}.go.plush` in the same folder.

Note that I've scavenged @Pantani's work from his PR #2603 :relaxed:

Reply:
I forgot to add a rationale about that in the description.

Reply:
Ok, I see. I'm just afraid of making the command too verbose by default, the more code generation we add, the more the user must understand when running the commands to generate an IBC module for example. This is also additional code to maintain.
I'm wondering if it is not possible to provide an empty stub for `ChannelKeeper` and `PortKeeper` from IBCkeeper used to initialize the module keeper. `testutil` is used as a util method to create the module keeper for the generate keeper tests, but the generated keeper tests only test the generated keeper methods, it is not necessary that IBC is initialized

Reply:
OK I'll check if I can reduce the number of loc.

Reply:
@lubtd I removed this file in favor of module specific stubs [7808101](https://github.com/ignite/cli/pull/2628/commits/7808101d006549473de6f47950f9d309bdcfc9a0)

Note for `PortKeeper.BindPort()` method I had to return a non-nil `Capability` or else one test fails.